env0/data_module.go

```diff
@@ -0,0 +1,83 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataModule() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataModuleRead,
+
+		Schema: map[string]*schema.Schema{
+			"module_name": {
+				Type:         schema.TypeString,
+				Description:  "the name of the module",
+				Optional:     true,
+				ExactlyOneOf: []string{"module_name", "id"},
+			},
+			"id": {
+				Type:         schema.TypeString,
+				Description:  "id of the module",
+				Optional:     true,
+				ExactlyOneOf: []string{"module_name", "id"},
+			},
+			"module_provider": {
+				Type:        schema.TypeString,
+				Description: "the provider name in the module source",
+				Computed:    true,
+			},
+			"repository": {
+				Type:        schema.TypeString,
+				Description: "template source code repository url",
+				Computed:    true,
+			},
+			"token_id": {
+				Type:        schema.TypeString,
+				Description: "the token id used for integration with GitLab",
+				Optional:    true,
+			},
+			"token_name": {
+				Type:        schema.TypeString,
+				Description: "the token name used for integration with GitLab",
+				Optional:    true,
+			},
+			"github_installation_id": {
+				Type:        schema.TypeInt,
+				Description: "The env0 application installation id on the relevant github repository",
```

Comment:
Please stick to lowercase attribute descriptions

Reply:
https://github.com/env0/terraform-provider-env0/pull/279/commits/a59b4bbbb4c4f0f861546b4848b2c55e5040adbb